tests/README.md

```diff
@@ -434,6 +434,13 @@ the test driver machine.
 
         $ cd kafka
         $ ./vagrant/aws/aws-init.sh
+
+* Add your AWS keys and load them (https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/):
```

Comment:
Hmm, so I think this might depend on where you're running from.

If you're running from a local machine and working with AWS instances, then this is probably right. (We've found that the latency and throughput when syncing files are too slow to make this very useful, though for just Kafka it is better.) If you're in AWS, then either you need to do this or you provision your driver VM with an IAM role that has these permissions and use the https://github.com/apache/kafka/blob/trunk/vagrant/aws/aws-access-keys-commands script to automatically pull in credentials (and these are auto-rotated so they have a limited window of vulnerability compared to your normal access keys).

I think the linked `aws-access-keys-commands` script isn't even the best one we have since it requires setting `AWS_IAM` as an env var, but I think you can get that automatically -- I think Confluent has the required script buried somewhere in its own bootstrap scripts.

Up to you how simple/complicated this gets...